core/statistics.c

```diff
@@ -207,6 +244,24 @@ void calculate_stats_summary(struct stats_summary *out, bool selected_only)
 		prev_month = current_month;
 		prev_year = current_year;
 	}
+
+	/* add labels for depth ranges up to maximum depth seen */
+	if (out->stats_by_depth[0].selection_size) {
+		d_idx = out->stats_by_depth[0].max_depth.mm;
+		if (d_idx > STATS_MAX_DEPTH * 1000) { d_idx = STATS_MAX_DEPTH * 1000; }
+		for (r=0; r * (STATS_DEPTH_BUCKET*1000) < d_idx; ++r) {
```

Comment:
Here and below:
Coding style: space before and after operators (`+`, `*`, `=`).